plugins/convert/scaling/_base.py

```diff
@@ -48,12 +48,12 @@ def run(self, new_face):
         reinsert_mask = False
         if new_face.shape[2] == 4:
             reinsert_mask = True
-            final_mask = new_face[:, :, -1]
-            new_face = new_face[:, :, :3]
+            final_mask = new_face[..., 3:4]
+            new_face = new_face[..., :3]
         new_face = self.process(new_face)
-        new_face = np.clip(new_face, 0.0, 1.0)
+        np.clip(new_face, 0.0, 1.0, out=new_face)
         if reinsert_mask and new_face.shape[2] != 4:
             # Reinsert Mask
-            new_face = np.concatenate((new_face, np.expand_dims(final_mask, axis=-1)), -1)
```

Comment:
final_mask = new_face[..., 3:4] instead of final_mask = new_face[..., 4] solves this